sales.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Salmon Cookies</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <section id="">
```

Comment:
If you aren't going to give it an id, just don't put in the id attribute at all.